packages/volto/src/components/manage/Sidebar/ObjectBrowserNav.jsx

```diff
@@ -135,10 +138,22 @@ const ObjectBrowserNav = ({
                   }
                   trigger={
                     <span>
-                      <Icon
-                        name={getContentIcon(item['@type'], item.is_folderish)}
-                        size="24px"
-                      />
+                      {item['@type'] === 'Image' ||
+                      item.hasPreviewImage === true ||
+                      item?.image_scales?.image?.length > 0 ? (
+                        <Image
```

Comment:
@calinvladth The Image component already takes care of what `getPreviewImageIcon` does. And you can force it to any scale you want.

https://6.docs.plone.org/volto/development/images.html

I think we can get along with it without PreviewImage component.